localstack/services/events/models_v2.py

```diff
@@ -94,3 +94,11 @@ class ValidationException(ServiceException):
     code: str = "ValidationException"
     sender_fault: bool = True
     status_code: int = 400
+
+
+class InternalInvalidEventPatternException(Exception):
```

Comment:
Why do we need to duplicate this exception?
Could we import the `InvalidEventPatternException` from the event_ruler as `InternalInvalidEventPatternException`?

Reply:
fixed in: https://github.com/localstack/localstack/pull/10664/commits/81d63c0bcf34bcf70377b1858bb736165682c209